.github/workflows/azure_kubernetes_aks_single_region_tests.yml

```diff
@@ -0,0 +1,588 @@
+              with:
+                  url: ${{ secrets.VAULT_ADDR }}
+                  method: approle
+                  roleId: ${{ secrets.VAULT_ROLE_ID }}
+                  secretId: ${{ secrets.VAULT_SECRET_ID }}
+                  exportEnv: false
+                  secrets: |
+                      secret/data/products/infrastructure-experience/ci/common AZURE_CLIENT_ID;
+                      secret/data/products/infrastructure-experience/ci/common AZURE_TENANT_ID;
+                      secret/data/products/infrastructure-experience/ci/common AZURE_SUBSCRIPTION_ID;
+
+            - name: Configure AWS CLI
+              uses: ./.github/actions/aws-configure-cli
+              with:
+                  vault-addr: ${{ secrets.VAULT_ADDR }}
+                  vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+                  vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+                  aws-profile: ${{ env.AWS_PROFILE }}
+                  aws-region: ${{ env.AWS_REGION }}
+
+            - name: Azure Login with OIDC
+              uses: azure/login@cb79c773a3cfa27f31f25eb3f677781210c9ce3d # v1.6.1
+              with:
+                  client-id: ${{ steps.secrets.outputs.AZURE_CLIENT_ID }}
+                  tenant-id: ${{ steps.secrets.outputs.AZURE_TENANT_ID }}
+                  subscription-id: ${{ steps.secrets.outputs.AZURE_SUBSCRIPTION_ID }}
+
+            - name: Set current Camunda version
+              id: camunda-version
+              run: |
+                  set -euo pipefail
+                  CAMUNDA_VERSION=$(cat .camunda-version)
+                  echo "CAMUNDA_VERSION=$CAMUNDA_VERSION" | tee -a "$GITHUB_OUTPUT"
+
+            - name: Generate resource names
+              id: generate_names
+              shell: bash
+              run: |
+                  set -euo pipefail
+
+                  PREFIX="${INPUT_RESOURCE_PREFIX:-}"
+                  CLUSTER="${INPUT_CLUSTER_NAME:-}"
+                  RG="${INPUT_RESOURCE_GROUP_NAME:-}"
+
+                  if [[ -z "$PREFIX" ]]; then
+                    PR_NUMBER=0
+                    if [[ "${GITHUB_EVENT_NAME}" == "pull_request" ]]; then
+                      # for pull_request events, pull in the PR number
+                      PR_NUMBER="$(jq --raw-output .pull_request.number < "$GITHUB_EVENT_PATH")"
+                    fi
+                    SFX="$(head /dev/urandom | tr -dc 'a-z0-9' | head -c5)"
+                    PREFIX="camunda-${PR_NUMBER}-${SFX}"
+                  fi
+
+                  if [[ -z "$CLUSTER" ]]; then
+                    CLUSTER="${PREFIX}-aks"
+                  fi
+                  if [[ -z "$RG" ]]; then
+                    RG="${PREFIX}-rg"
+                  fi
+
+                  {
+                    echo "RESOURCE_PREFIX=${PREFIX}";
+                    echo "CLUSTER_NAME=${CLUSTER}";
+                    echo "RESOURCE_GROUP_NAME=${RG}"
```

Comment:
just as reference https://www.shellcheck.net/wiki/SC2129
don't need to change it, just FYI that `;` isn't required.